src/drivers/fluid_waveout.c

```diff
@@ -36,12 +36,48 @@
 /* Milliseconds of a single sound buffer */
 #define MS_BUFFER_LENGTH    20
 
+/**
+* The driver handle multiple channels.
+* Actually the number maximum of channels is limited to  2 * WAVEOUT_MAX_STEREO_CHANNELS.
+* The only reason of this limitation is because we dont know how to define the mapping
+* of speakers for stereo output number above WAVEOUT_MAX_STEREO_CHANNELS.
+*/
+/* Maximum number of stereo outputs */
+#define WAVEOUT_MAX_STEREO_CHANNELS 4
+
+/* speakers mapping */
+static DWORD channel_mask_speakers[WAVEOUT_MAX_STEREO_CHANNELS] =
```

Comment:
This array is a duplicate from dsound. Would it make sense  to remove the `static` qualifier and simply reuse the `channel_mask` from the dsound?

Reply:
Both dsound and waveout are independent cmake option.

Reply:
Oh, right. Ok then, never mind.